### PR TITLE
EVA-987 Update Documentation and add CircleCI to tesserae

### PR DESCRIPTION
**Pull request by tylerwilding-wk**

### CODE REVIEW

**Comment by aviary3-wk** [comment]
## Security Insights
The items listed below may not capture all security relevant changes. Before providing a security review, be sure to review the entire PR for security impact.
<details>
<summary>(1) Security relevant changes were detected</summary>
<li>Watched keyword <code>.innerHTML =</code> in <code>documentation/java/script.js</code> line(s) ['23', '27'] added</li>
</details>

## Action Items
* Obtain a [security review](https://wiki.atl.workiva.net/display/SECURITY/Development+Security+Review+Guidelines); reviewer should pay special attention to insights listed above
* Verify [`aviary.yaml`](https://dev.workiva.net/docs/teams/information-security/aviary#configuring-aviary) coverage of security relevant code
---
_Questions or Comments? Reach out on Slack: #support-infosec._

File: README.md [hunk 1]
```diff
@@ -302,16 +327,12 @@ Register a function by calling `watch`. For cases in which a callback is potenti
...
 
-## Contributing
+## Maintainers and Contributors
 
-1. Branch and PR to master
-2. Maintainers will review.
+### Active Maintainers
 
-Guidelines:
+-
```
Comment (danielharasymiw-wf) [inline README.md]: Do we not have anyone to add to this list? :(

**Comment by danielharasymiw-wf** [comment]
+1
security +1

File: README.md (hunk 1 above)
Comment (tylerwilding-wk) [reply README.md]: No one explicitly yet, we can revisit and update these once everything is wrapped up.

**Comment by tylerwilding-wk** [comment]
@Workiva/release-management-p

**Pull request merged**